.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.0
+
+jobs:
+  "Test":
+    docker: # Replace this with a fastlane owned docker image at some point
+      - image: joshdholtz/fastlane-docs:0.0.1
+    environment:
```

Comment:
Could this maybe be moved to the docker image so it doesn't have to be repeated here?

Reply:
Can do can do!